clang/lib/CConv/ConstraintResolver.cpp

```diff
@@ -513,6 +515,17 @@ void ConstraintResolver::constrainLocalAssign(Stmt *TSt, Expr *LHS, Expr *RHS,
   std::set<ConstraintVariable *> L = getExprConstraintVars(LHS);
   std::set<ConstraintVariable *> R = getExprConstraintVars(RHS);
   constrainConsVarGeq(L, R, Info.getConstraints(), &PL, CAction, false, &Info);
+
+  // Only if all types are enabled and these are not pointers, then track
+  // the assignment.
+  if (AllTypes && L.empty() && R.empty()) {
```

Comment:
I don't know if this is right. If `L` or `R` are not pointers you should still get a CV back, i.e., see function `ConstraintResolver::PVConstraintFromType` and also `ConstraintResolver::getBaseVarPVConstraint` which are called from `ConstraintResolver::getExprConstraintVars(Expr *E)`